contracts/levels/Switch.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.4.24;
+
+contract Switch {
+  bool public switchOn; // switch is off
+  bytes4 public offSelector = bytes4(keccak256("turnSwitchOff()"));
+
+  function flipSwitch(bytes _data) public

```

Comment:
Maybe format the modifier on the same line, like you do on the others, so as to not draw too much attention to it (and for consistency's sake)?

Reply:
yep, nice catch.